bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/FileAudioStream.java

```diff
@@ -70,6 +75,36 @@ private static AudioFormat getAudioFormat(File file) throws AudioException {
         }
     }
 
+    private static AudioFormat parseWavFormat(File file) {
```

Comment:
It looks like this method is useful for `ByteArrayAudioStream` too (see https://github.com/openhab/openhab-addons/pull/11877). Maybe we should move it to a different class to be available as util method.

Reply:
Yes, good idea. Done